dev-libs/miniaudio/miniaudio-0.11.11.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A single file library for audio playback and capture."
+HOMEPAGE="https://miniaudio.io"
+COMMIT="a0dc1037f99a643ff5fad7272cd3d6461f2d63fa"
+SRC_URI="https://github.com/mackron/miniaudio/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${COMMIT}"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+src_install() {
+	insinto /usr/include/${PN}/
+	doins -r *
+}
+
+src_test() {
+	cd "${BUILD_DIR}/test" || die
```

Comment:
BUILD_DIR isn't defined and this doesn't work for me. Did you definitely run the tests w/ "ebuild ... clean test install" or FEATURES=test?

Reply:
ah.. no.. I have not tried any tests.
Sorry. I just left it there.

Reply:
can we leave out those tests?

Reply:
Yeah it's header only and no build system being used right now so it's ok. Can revisit another time

Reply:
should I remove test then?